README.rst

```diff
@@ -99,6 +102,7 @@ And here is how you can configure different versions:
         'title'                => 'Symfony2 API',
         'build_dir'            => __DIR__.'/../build/sf2/%version%',
         'cache_dir'            => __DIR__.'/../cache/sf2/%version%',
+        'remote_repository'    => new GitHubRemoteRepository('symfony/symfony', dirname($dir)),
```

Comment:
The comment above says `The ``Sami`` constructor optionally takes an array of options as a second argument:` which kind of implies that all options are optional. Any ideas on how we can stress that fact even more, it's even needed?